ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -75,7 +76,7 @@ import           Ouroboros.Storage.LedgerDB.InMemory (ledgerDbDefaultParams)
 --
 -- This function runs forever unless an exception is thrown.
 run
-  :: forall blk peer. (RunNode blk, Ord peer)
+  :: forall blk peer. (RunNode blk, Ord peer, Show peer)
```

Comment:
Why do we need this `Show peer`? Typically, using a `Tracer` avoids these constraints, as they only need to be satisfied by the `Tracer`, not by every use site of the it.

Reply:
It is used when constructing an identifying tag for the Mux bearer.

Reply:
@karknu I found it strange too, maybe we can address this in the future.

Reply:
I opened an issue for this: https://github.com/input-output-hk/ouroboros-network/issues/1120